scripts/stage1.sh

```diff
@@ -29,7 +29,7 @@ function get_deps() {
 	#
 	sudo rm /etc/ld.so.cache
 	sudo ldconfig
-	(LD_LIBRARY_PATH="$(ldconfig -v 2>/dev/null | grep -v ^$'\t' | sed "s@^@$workdir@" | tr -d '\n'):$workdir/lib/systemd" \
+	(LD_LIBRARY_PATH="$(ldconfig -v 2>/dev/null | grep -v ^$'\t' | sed "s@^@$workdir@" | cut -d':' -f1 | paste -sd':' | tr -d '\n'):$workdir/usr/lib/x86_64-linux-gnu/systemd" \
```

Comment:
Do we need both the `tr -d '\n'` and the `paste -sd':'`? Those both serve to combine everything onto one line, right?

Reply:
fixed